test/model/message_test.dart

```diff
@@ -269,6 +271,80 @@ void main() {
     test('rendering-only update does not change timestamp (for old server versions)', () async {
       await checkRenderingOnly(legacy: true);
     });
+
+    group('Handle message edit state update', () {
+      final message = eg.streamMessage();
+      final otherMessage = eg.streamMessage();
+
+      Future<void> sendEvent(Message message, UpdateMessageEvent event) async {
+        await prepare();
+        await prepareMessages([message, otherMessage]);
+
+        await store.handleEvent(event);
+        checkNotifiedOnce();
+      }
+
+      test('message not moved update', () async {
+        await sendEvent(message, eg.updateMessageEditEvent(message));
+        check(store).messages[message.id].editState.equals(MessageEditState.edited);
+        check(store).messages[otherMessage.id].editState.equals(MessageEditState.none);
+      });
+
+      test('message topic moved update', () async {
+        await sendEvent(message, eg.updateMessageMoveEvent([message, otherMessage],
+          origTopic: 'old topic',
+          newTopic:  'new topic',
+        ));
```

Comment:
Can save a line:
```suggestion
        await sendEvent(message, eg.updateMessageMoveEvent([message, otherMessage],
          origTopic: 'old topic',
          newTopic:  'new topic'));
```
(here and below)